cli/src/commands/debug/init_local.rs

```diff
@@ -34,17 +34,17 @@ use crate::ui::Ui;
 /// does not exist, it will be created. If no directory is given, the current
 /// directory is used.
 #[derive(clap::Args, Clone, Debug)]
-pub(crate) struct DebugInitLocalArgs {
+pub(crate) struct DebugInitSimpleArgs {
```

Comment:
nit: update the rustdoc comment too. i'm fine with just deleting the line about why it's called "local"

Reply:
done